notification/irc.py

```diff
@@ -63,6 +63,10 @@
     description:
       - Channel name
     required: true
+  nick_to:
+    description:
+      - Nick to send the message to
+    required: false
```

Comment:
* Need to add version_added: 2.0
* Probably should update the nick: documentation as well so that people know that nick is the irc nickname being sent from.
* Depending on what you decide below about the relationship between nick and channel, need to add information about that to the documentation here.